docker-compose.yml

```diff
@@ -33,9 +33,28 @@ services:
 #      - cluster.routing.allocation.disk.threshold_enabled=false
 #    ports:
 #      - 9200:9200
-#  redis:
-#    # NOTE: No persistance storage configured.
-#    # See: https://hub.docker.com/_/redis/
-#    image: redis
-#    ports:
-#        - 6379:6379
+
+  redis:
+    # NOTE: No persistance storage configured.
+    image: redis
+
+  celery:
+    build: .
+    environment: &celery_env
```

Comment:
you'll probably want to register that as `&web_env` and re-use that - since the web should also know which broker is used etc.

Reply:
done